airbyte-integrations/connectors/destination-selectdb/src/main/resources/spec.json

```diff
@@ -0,0 +1,54 @@
+{
+  "documentationUrl": "https://docs.airbyte.com/integrations/destinations/selectdb",
+  "supportsIncremental": false,
+  "supportsNormalization": false,
+  "supportsDBT": false,
+  "supported_destination_sync_modes": ["append", "overwrite"],
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "SelectDB Destination Spec",
+    "type": "object",
+    "required": [
+      "load_url", "jdbc_url", "cluster_name", "user_name", "password", "database"
+    ],
+    "properties": {
+      "load_url": {
+        "title": "loadURL",
+        "description": "load host and port: xxx.privatelink.aliyun.com:47057",
+        "type": "string",
+        "order": 0
+      },
+      "jdbc_url": {
+        "title": "jdbcURL",
+        "description": "jdbc host and port: xxx.privatelink.aliyun.com:30523",
+        "type": "string",
+        "order": 1
+      },
+      "cluster_name": {
+        "title": "ClusterName",
+        "description": "clusterName of SelectDB",
+        "type": "string",
+        "order": 2
+      },
+      "user_name": {
```

Comment:
change to username

Reply:
selectdb account user is email address, database user is 'admin'(eg: mysql -u admin)

Reply:
username is 'admin' (eg:mysql -u admin ), selectdb account username is email address